backend/postgres/postgres.go

```diff
@@ -62,6 +63,12 @@ type config struct {
 	// but idle connections are still discarded by the client
 	// if IdleTimeout is set.
 	IdleCheckFrequency time.Duration `env:"POSTGRES_IDLE_CHECK_FREQUENCY" envDefault:"1m"`
+	// Name of the Table that is created to try if database is writeable
+	HealthTableName string `env:"POSTGRES_HEALTHCHECK_TABLE" envDefault:"healthcheck"`
```

Comment:
Maybe streamline naming here and for `HealthMaxRequest` and `HealthWrite`. Example: 
```go
HealthCheckTableName string `env:"POSTGRES_HEALTH_CHECK_TABLE_NAME" envDefault:"healthcheck"`
```